src/pages/base/settings.js

```diff
@@ -57,6 +58,15 @@ export default class BaseSettingsPage extends React.Component {
       name = `${user.more.firstName} ${user.more.lastName}`;
     }
 
+    let saveButton;
+    if (!_.endsWith(window.location.href, 'followers')) {
```

Comment:
please do the check basing on some props, forwarded from outer component. relying on location is too fragine